src/WCSimWCTrigger.cc

```diff
@@ -406,6 +409,210 @@ void WCSimWCTriggerBase::FillDigitsCollection(WCSimWCDigitsCollection* WCDCPMT,
 }
 
 
+void WCSimWCTriggerBase::FillDigitsCollectionNoTrigger(WCSimWCDigitsCollection* WCDCPMT, bool remove_hits, TriggerType_t save_triggerType)
```

Comment:
Great - combining will make it easier to maintain & less error prone. Things to be careful of (@stewartt1982 please add if I've missed everything)
* Skip the SaveFailures stuff for NoTrigger
* Make sure the trigger time window is large enough (this should be done by adding NoTrigger cases to GetPreTriggerWindow() and GetPostTriggerWindow())
* Multiple digits per trigger - ensure they are always allowed when using NoTrigger
* Offset - turn this off when using NoTrigger (or leave it on - not sure what the purpose of turning it off is)